src/resources/elements/dropdown/dropdown.scss

```diff
@@ -1,126 +1,74 @@
 @import "styles/styles.scss";
 
-@mixin defaultTransition($properties...) {
-  $transition: ();
-  @each $property in $properties {
-    $transition: append($transition, ($property 0.16s ease-in-out), $separator: comma);
-  }
-  -webkit-transition: $transition;
-  -moz-transition: $transition;
-  -ms-transition: $transition;
-  -o-transition: $transition;
-  transition: $transition;}
-
 .dropdown {
-  border: 1px solid $Border01;
-  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.25);
-  background: $BG01;
   color: $Neutral02;
-  border: 1px solid $Border01;
-  border-radius: 10px;
-  font-family: Inter;
-  width: fit-content;
-  height: fit-content;
+  transition: all 300ms;
   position: relative;
-  &:focus-within, &:active, &.menuShowing {
-    border-color: $Secondary02;
+  width: 200px;
+  .selectedLabel, .menu {
+    cursor: pointer;
+    border-radius: 10px;
+    border: 1px solid #403453;
+    box-sizing: border-box;
+    width: 100%;
   }
-
-  @include defaultTransition(border-radius, box-shadow, border-color);
-
-  > .selectedLabel {
+  .selectedLabel {
     display: flex;
+    background: $BG01;
     justify-content: space-between;
-    cursor: pointer;
-    margin: 11px;
-    margin-right: 0;
-
-    .itemContent {
-      margin-right: 8px;
-    }
-
-    div.el-icon-arrow {
-      margin-right: 0.7em;
-      transition: all 0.3s;
+    align-items: center;
+    height: 46px;
```

Comment:
I've removed the height and added line-height instead to match with the other input-field styling. Also added white-space and text-overflow, to prevent text leaking out of the container